helpers/emulation.js

```diff
@@ -92,8 +92,4 @@ function enableNetworkThrottling(driver) {
   return driver.sendCommand('Network.emulateNetworkConditions', TYPICAL_MOBILE_THROTTLING_METRICS);
 }
 
-module.exports = {
-  enableNexus5X: enableNexus5X,
-  enableNetworkThrottling: enableNetworkThrottling,
-  disableCache: disableCache
-};
+module.exports = {enableNexus5X, enableNetworkThrottling, disableCache};
```

Comment:
not sure how others feel, but I actually prefer each item on its own line, for easy reading